src/models/SetCovering.jl

```diff
@@ -21,6 +21,7 @@ Base.:(==)(a::SetCovering, b::SetCovering) = a.sets == b.sets && a.weights == b.
 
 #variables interface
 variables(gp::SetCovering) = [1:length(gp.sets)...]
+num_variables(gp::SetCovering) = length(gp.sets)
```

Comment:
`num_variables` is already generally defined in `src/models/models.jl`, so we might not need redefine this for SetCovering.